src/components/Headline/StyledDiaryHeadline.js

```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components';
```

Comment:
Same as above - the name of your file suggests that there are only styled-components in here but this is actually a full component (see function starting from line 16).